### PR TITLE
FIX: Restore D213 in flake8

### PR DESCRIPTION
**Pull request by timhoffm**
This got accidentally ignored in #25147.

We want D213 (#17125, #17193)
https://matplotlib.org/devdocs/devel/document.html#quote-positions

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -18,7 +18,9 @@ ignore =
     D301,
     D400, D401, D403, D404
     # ignored by pydocstyle numpy docstring convention
-    D107, D203, D212, D213, D402, D413, D415, D416, D417,
+    D107, D203, D212, D402, D413, D415, D416, D417,
+    # while D213 is also part of numpy docstring convention, it's left out above,
```
Comment (rcomer) [inline .flake8:22]:
```suggestion
    # while D213 is also ignored by numpy docstring convention, it's left out above,
```
I may have misunderstood, but the [numpydoc styleguide](https://numpydoc.readthedocs.io/en/latest/format.html) doesn't seem to care about this rule.

Comment (timhoffm) [reply .flake8]: Yes, thanks.

**Review by rcomer — APPROVED** [review]

**Comment by tacaswell** [comment]
The py39 osx failure looks like an install failure (that I can not get my github and work msft accounts to play nice to allow me to restart...)

**Review by tacaswell — APPROVED** [review]
Anyone can merge on green.

**Comment by rcomer** [comment]
I also got "you don't have permissions" when I tried to restart the failed test within Azure, which is odd as I'm sure I did it with the same MS account a couple of days ago.  However, I found a button within GitHub that I hadn't spotted before.  It's in the sidebar after you click on one of the appveyor task "details" links.

image

**Comment by QuLogic** [comment]
i've never set up an account for Azure, but the button on the sidebar there has always worked for me, so it's probably a better choice overall.

**Pull request merged**